File: TellDontAsk/AlarmsProcedural/AlarmServices/LoudAlarmService.cs
using System;
using AlarmsProcedural.Adapters;

namespace AlarmsProcedural.AlarmServices
{
  public static class LoudAlarmService
  {
    public static void Trigger()
    {
      SirensService.PlayLoudSound();
    }

    public static void Disable()
    {
      SirensService.StopPlayingLoudSound();
    }

    public static void Dump(Alarm alarm)
    {
      Console.WriteLine("{ Playing loud sound }");
    }
  }
}

File: Java/AlarmsProcedural/src/main/java/AlarmServices/LoudAlarmService.java
package AlarmServices;

import Adapters.SirensService;
import Main.Alarm;

public class LoudAlarmService {
    public static void trigger() {
      SirensService.playLoudSound();
    }

    public static void disable() {
      SirensService.stopPlayingLoudSound();
    }

    public static void dump(Alarm alarm) {
      System.out.println("{ Playing loud sound }");
    }
  }
